test/frontdesk_test.rb

```diff
@@ -0,0 +1,161 @@
+require_relative 'test_helper'
+
+describe "FrontDesk class" do
+  before do
+    @range = Hotel::DateRange.new(start_date: "1st Apr 2020", end_date: "3rd Apr 2020")
+    @front_desk = Hotel::FrontDesk.new
+  end
+
+  describe "#initialize" do
```

Comment:
These tests are very thorough. Nice work! You might consider including some of these assertions in a single `it` block. For instance `it "has 20 items in a rooms list" do` could include the following 2 assertions:
```
expect(@front_desk.rooms.length).must_equal 20
expect(@front_desk.rooms.last).must_be_instance_of Hotel::Room
````